README.md

```diff
@@ -47,6 +47,10 @@ TiP will do this only once - the first time user completes BCP.
 You can also use TiP if your web app is load balanced across multiple instances. In this case TiP will 
 trigger once per instance, thus if you have 3 instances, then GitHub API will be hit six times, however 
 once a label is set on the PR, trying to set it again has no ill effect.
+
+TiP is designed to minimise risk for teams that operate under rapid continuous deplyment culture where deployments are straight to production and without much testing:
```

Comment:
What about: "TiP is designed to mitigate risk in a continuous deployment workflow, by providing fast feedback on production code."